allensdk/brain_observatory/ecephys/probes.py

```diff
@@ -131,10 +114,9 @@ def get_units_table(
                 ) for p in self.probes
             ])
 
-            structure_key = _get_structure_key(channels)
-
             if filter_out_of_brain_units:
-                channels = channels[~(channels[structure_key].isna())]
+                channels = channels[
+                    ~(channels['manual_structure_acronym'].isna())]
```

Comment:
I have no idea if this is tested or not, but this block of code

https://github.com/AllenInstitute/AllenSDK/blob/master/allensdk/brain_observatory/ecephys/ecephys_project_api/ecephys_project_lims_api.py#L273-L274

leads me to think that we will need to support the use of `'ecephys_structure_acronym'`

Reply:
As I recall, the need to support `'ecephys_structure_id'` was revealed by the on-prem test. Maybe that will tell us here, as well.

Reply:
The bit of code that references ecephys_structure_id above is from a LIMS query, which is separate from reading/writing NWB files. I still think this ID is internal to LIMS and should not be returned to external users.

Reply:
I'm not disputing that we should obfuscate the id. I'm worried that VCN-generated dataframes will use the column `ecephys_structure_acronym` where we use `manual_structure_acronym`.

The code you removed that scanned dataframes to see if they uses `ecephys_structure_id` or `manual_structure_id` was to provide a similar backward compatibility. I was referencing that block of code above to show that some previous version of this team aliased `manual_structure_acronym` as `ecephys_structure_acronym`. We will likely need to do something about that (either by changing the code that does the aliasing, or doing something here to detect which column name is current).

Reply:
VCN uses this same code path to read from json. We are reusing that write_nwb schema where this acronym is referred to as `manual_structure_acronym`. So the VCN write nwb also expects this name. The other code in the project_lims_api is independent from this code.

Reply:
The VCN code created columns called ecephys_structure_acronym and ecephys_structure_id [here](https://github.com/AllenInstitute/AllenSDK/blob/master/allensdk/brain_observatory/ecephys/ecephys_session_api/ecephys_nwb_session_api.py#L144), so yes, we will be changing the channels table. I think a user who was relying on a field called `ecephys_structure_acronym` can rename the column `manual_structure_acronym`. Or if you feel strongly about it, we can rename the VCN column in its api.

Reply:
Okay. I've convinced myself that you are right. Thanks for entertaining this.